spec/graphql/analysis/ast/max_query_depth_spec.rb

```diff
@@ -131,4 +131,20 @@
       assert_equal "Query has depth of 7, which exceeds max depth of 4", result.message
     end
   end
+
+  describe "when the query would cause a stack error" do
+    let(:query_string) {
+      str = "query { cheese(id: 1) { ".dup
+      n = 10_000
+      n.times { str << "similarCheese(source: SHEEP) { " }
+      str << "id "
+      n.times { str << "} " }
+      str << "} }"
+      str
+    }
+
+    it "returns an error" do
+      assert_equal ["This query is too large to execute."], query.static_errors.map(&:message)
```

Comment:
I'm assuming this is meant to end up in the result? The ends up as `{"errors"=>[]}` though

https://github.com/rmosolgo/graphql-ruby/blob/da513ac4a3ad3b2d86587180474ae9f96863bc41/lib/graphql/execution/multiplex.rb#L128

At this point, `static_errors` (and `analysis_errors`) appear to be reset back to `[]` but I couldn't track down where that happens.

Reply:
🤦